src/tests/ftest/launch.py

```diff
@@ -2335,11 +2388,6 @@ def main():
         "-c", "--clean",
         action="store_true",
         help="remove daos log files from the test hosts prior to the test")
-    parser.add_argument(
-        "-d", "--discard",
-        action="store_true",
-        help="when replacing server/client yaml file placeholders, discard "
-             "any placeholders that do not end up with a replacement value")
```

Comment:
Just a thought - Should we keep this option but just not use it, for the sake of backwards compatibility with peoples' scripts? Maybe just log a deprecated message?